core/common/src/TerrainSettings.ts

```diff
@@ -80,11 +105,31 @@ export class TerrainSettings {
     return this._nonLocatable;
   }
 
-  constructor(providerName: string = "CesiumWorldTerrain", exaggeration: number = 1.0, applyLighting = false, heightOrigin = 0.0, heightOriginMode = TerrainHeightOriginMode.Geodetic) {
-    this.providerName = providerName;
-    this.exaggeration = Math.min(100, Math.max(0.1, exaggeration));
-    this.applyLighting = applyLighting;
-    this.heightOrigin = heightOrigin;
+  /** @deprecated in 4.x. Use the overload that takes [[TerrainProps]]. */
```

Comment:
```suggestion
  /** @deprecated in 4.5.x. Use the overload that takes [[TerrainProps]]. */
```

Reply:
i think adding the minor is useful. i _believe_ if an api was deprecated in 4.0, you could remove it in 5.0, where as if it was deprecated in 4.1 it cant be removed til 6.0